currency_cost.py

```diff
@@ -0,0 +1,46 @@
+"""
+Cost object for currency
+"""
+
+import collections
+
+class CurrencyCost:
+    """
+    Save information in a currency trading.
+    """
+    def __init__(self, region, instruments):
```

Comment:
While `region` means where our trading center is, base currency is another issue.
For example, we can trade in Australia, but how we evaluate our PnL, does not need to be based on AUD.
We can use USD as our base currency, or whatever we like.
Given that `region` is not really important when we calculate PnL, I suggest to rename this variable to `base`, and its value should always be some currency, instead of region code. (USD vs US)